src/dataplane-protocol/src/record.rs

```diff
@@ -331,6 +332,17 @@ where
     }
 }
 
+impl<'a, B> From<&'a [u8]> for Record<B>
```

Comment:
this is not optimum but it's not show stopper. we should create separate issue to optimize for sending bytes